src/store/dappChainStore.js

```diff
@@ -349,25 +350,22 @@ export default {
       const network = state.chainUrls[state.chainIndex].network
       let user 
       try { 
-        user = await DPOSUser.createEthSignMetamaskUserAsync(		
-          rootState.DPOS.web3,
-          getters.dappchainEndpoint,
-          network,
-          GW_ADDRESS || GatewayJSON.networks[network].address,
-          LOOM_ADDRESS || LoomTokenJSON.networks[network].address
-        )
+        user = await DPOSUserV3.createEthSignMetamaskUserAsync({
```

Comment:
This is part of initializing the DPOSUser class in V3 with an interface, https://github.com/loomnetwork/loom-js/pull/263/commits/8703764b7d79b9bf2c8daa7270a7d1b3b1d25313#diff-612dbd618dea0c56bcb21059a1bf585eR49